includes/admin/class-wc-rest-stripe-account-keys-controller.php

```diff
@@ -13,8 +13,6 @@
  * @since 5.6.0
  */
 class WC_REST_Stripe_Account_Keys_Controller extends WC_Stripe_REST_Base_Controller {
-	const STRIPE_GATEWAY_SETTINGS_OPTION_NAME = 'woocommerce_stripe_settings';
```

Comment:
We shouldn't remove this public constant; merchant custom code may depend on it.

Reply:
Reverted it and added a comment above 👍